docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,222 @@
+
+* *Convenient management and flexibility:* Compared to SQLite, FBC offers a more convenient solution. Rather than adding one set of graph updates at a time, users can define all updates in one place using File-based catalogs. This centralized management capability simplifies the process and improves overall efficiency.
+
+* *Improved ownership and analysis:* FBC provides users with improved ownership and control over their graphs. They offer a clean and declarative template that facilitates comprehensive analysis and sanity checks. Users can easily determine upgrade possibilities and recover from error states, no longer relying on limited configuration options found in localized CSV files.
+
+
+For additional information, see link:https://olm.operatorframework.io/docs/reference/file-based-catalogs/[File based Catalog].
+
+== Setting up an FBC Compatible git repository
+
+An FBC Compatible Git repository has the following directory structure:
+
+----
+Project Root
+│
+├─ Dockerfile (for example, catalog.Dockerfile)
+│
+├─ Directory for the targetted OCP version (for example, 4.12)
+│   ├─ devfile.yaml
+│   └─ catalog
+        └─ catalog.yaml
+|
+├─ Directory for the targetted OCP version (for example, 4.13)
+│   ├─ devfile.yaml
+│   └─ catalog
+        └─ catalog.yaml
+│
+└─ README.md
+----
+
+
+* *`catalog.Dockerfile`*: This file contains the instructions for building the FBC image. It should specify a parent image, such as `registry.redhat.io/openshift4/ose-operator-registry`. The Dockerfile should also have a tag corresponding to the target OCP version. 
+
++	
+*Example:*
+
++
+[source,catalog.Dockerfile]
+----
+# The base image is expected to contain
+# /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
+FROM registry.redhat.io/openshift4/ose-operator-registry:v4.12
+
+# Configure the entrypoint and command
+ENTRYPOINT ["/bin/opm"]
+CMD ["serve", "/configs", "--cache-dir=/tmp/cache"]
+
+# Copy declarative config root into image at /configs and pre-populate serve cache
+ADD catalog /configs
+RUN ["/bin/opm", "serve", "/configs", "--cache-dir=/tmp/cache", "--cache-only"]
+
+# Set DC-specific label for the location of the DC root directory
+# in the image
+LABEL operators.operatorframework.io.index.configs.v1=/configs
+----
+
+* *`catalog.yaml`*: This file contains important metadata about the operator. There should be a `catalog.yml` file for each FBC image that is produced.
+
++
+*Example:*
+
++
+[source,yaml]
+----
+schema: olm.composite.catalogs
+catalogs:
+  - name: "4.13" <1>
+    destination:
+      baseImage: registry.redhat.io/openshift4/ose-operator-registry:v4.13 <2>
+      workingDir: .
+    builders:
+      - olm.builder.basic
+  - name: "4.12"
+    destination:
+      baseImage: registry.redhat.io/openshift4/ose-operator-registry:v4.12 This template can be hosted
```

Comment:
Is "this template can be hosted" an annotation, or part of the example file? It feels out of place as currently styled, but I could be misunderstanding the purpose.